src/main/java/og_spipes/service/SPipesExecutionService.java

```diff
@@ -88,7 +88,7 @@ private void createDebugConfig(String configLocation, String moduleScript){
 
         File file = new File(configLocation);
         try (FileWriter writer = new FileWriter(file)){
-            model.write(writer, "TTL");
+            model.write(writer, "TTL");  // TODO apply JenaUtils.writeScript(writer, ontModel);
```

Comment:
```suggestion
            model.write(writer, "TTL");  // TODO apply JenaUtils.writeScript(writer, ontModel);
```

I do not understand why (**maybe it is important**) here is writer chosen instead of outputstream. Maybe it would be good to find out how it works and if it works same with outputstream.

Reply:
@MSattrtand

Reply:
From the Jena documentation, the description for write( Writer writer, String lang ) : 

> Write a serialized representation of a model in a specified language. It is often better to use an OutputStream rather than a Writer, since this will avoid character encoding errors.

So, using OutputStream will avoid character encoding errors

Reply:
ok